### ISSUE TITLE
NVDA jumps to an invisible dialog  on chatgpt website

### ISSUE THREAD

**Issue by dreinn** [nvaccess/nvda#17594]
### Actual behavior:
go to 
https://chat.openai.com/
signin with your preferred method.  You'll see that NVDA fall out into a "dialog" and you can't write anything. You must press ctrl-nvda-space for exiting object. Then you can write, but when ChatGpt composes an answer, the cursor return to the invisible dialog.
Other screen reader doesn't have this behaviour.

### Expected behavior:
NVDA should ignore the invisible dialog.

#### Windows version: 
win10 and win11.

#### Name and version of other software in use when reproducing the issue: 
Chrome, Edge Firefox, Brave, all versions.
I'm aware that this is probably not a real NVDA bug, but as mentioned, all other screen readers ignore the dialog that appears, and I had to point it out somehow!

**Comment by amirsol81** [comment]
Yes, duplicable with the latest alphas. FYI it doesn't affect JAWS.

**Comment by SaschaCowley** [comment]
Welcome @dreinn ,

We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by CyrilleB79** [comment]
As a workaround, press shift+escape shortcut key to focus the input field of the chat.

**Comment by dreinn** [comment]
Now the issue is properly readable, there were 2 bad signs that prevent some information to appear. Sorry.

**Comment by gerald-hartig** [comment]
@dreinn Given that their website is constantly evolving, can you confirm that this is still an issue?

**Comment by dreinn** [comment]
@gerald-hartig try to imagine this situation:
I should write to openai, telling that NVDA has a problem on their website, but Jaws, Narrator and Voiceover works properly.
In your opinion, what kind of answer should I receive?
Any person will answer me to go back to the NVDA developers, don't you think?

Be that as it may, the problem is always present; NVDA ends up in an invisible dialog

**Comment by CyrilleB79** [comment]
@gerald-hartig, please note that ChatGPT is not the only site where NVDA has issues with website dialogs. I have at least two other in mind:
* GitHub's "Finish your review" dialog
* opening reCAPTCHA in some site (I still need to find again which one(s))
* and probably more example I do not recall specifically

I do not know if the websites honour coding and accessibility guidelines. But given the pattern is quite common and other screen reader support it, wouldn't it be worth for NVDA to look at it, I mean even if ChatGPT's website changes in the future?

It's worth noting that FF does not exhibit this issue. @jcsteh your opinion may be also valuable; maybe you have already thought to it / done something explicitly so that the bug be not present on FF?

**Comment by ABuffEr** [comment]
> @gerald-hartig, please note that ChatGPT is not the only site where NVDA has issues with website dialogs

With issues, do you mean specifically the need to escape from an invisible dialog?
Because, otherwise, I have other issues with dialogs (like nested ones on FB), even with FF unfortunately.

**Comment by SaschaCowley** [comment]
@CyrilleB79 said
> It's worth noting that FF does not exhibit this issue.

However, @dreinn  said
> Name and version of other software in use when reproducing the issue:
> Chrome, Edge Firefox, Brave, all versions.

I was unable to reproduce this issue yesterday, and don't have trouble using the finish review dialog on GitHub.
@dreinn is the ChatGPT issue present in Firefox?
@CyrilleB79 can you describe the problem you're talking about in more detail?

**Comment by dreinn** [comment]
@SaschaCowley  I just installed Firefox and I gave it a try, yes, the issue is confirmed.
It is important to be logged with the personal account on the website, because if you are not logged in, there are no problems.
I don't know if it can be useful for you, but this is the developer information about the object "dialog", with Chrome:

Developer info for navigator object:
name: None
role: DIALOG
processID: 23784
roleText: None
states: OFFSCREEN, FOCUSABLE, FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_WebDialogDocumentIa2WebIAccessible object at 0x0164C5D0>
Python class mro: (<class 'NVDAObjects.Dynamic_WebDialogDocumentIa2WebIAccessible'>, <class 'NVDAObjects.behaviors.WebDialog'>, <class 'NVDAObjects.IAccessible.chromium.Document'>, <class 'NVDAObjects.IAccessible.ia2Web.Document'>, <class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'garbageHandler.TrackedObject'>, <class 'object'>)
description: None
location: RectLTWH(left=0, top=151, width=1920, height=869)
value: None
TextInfo: <class 'NVDAObjects.IAccessible.IA2TextTextInfo'>
appModule: AppModule(appModuleHandler, appName='chrome', processID=23784)
appModule.productName: 'Google Chrome'
appModule.productVersion: '131.0.6778.265'
appModule.helperLocalBindingHandle: c_long(197675640)
appModule.appArchitecture: 'AMD64'
windowHandle: 723254
windowClassName: 'Chrome_RenderWidgetHostHWND'
windowControlID: 33584368
windowStyle: 1445986304
extendedWindowStyle: 32
windowThreadID: 15532
windowText: 'Chrome Legacy Window'
displayText: ''
IAccessibleObject: <POINTER(IAccessible2) ptr=0xbcd0924 at a920a30>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=723254, objectID=-4, childID=-4195
IAccessible accName: None
IAccessible accRole: ROLE_SYSTEM_DIALOG
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_OFFSCREEN, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1114116)
IAccessible accDescription: None
IAccessible accValue: ''
IAccessible2 windowHandle: 723254
IAccessible2 uniqueID: -4195
IAccessible2 role: ROLE_SYSTEM_DIALOG
IAccessible2 states: IA2_STATE_OPAQUE (1024)
IAccessible2 attributes: 'display:block;tag:div;xml-roles:dialog;id:radix-\\:R19njkkuiiklj5\\:;text-align:left;'
IAccessible2 relations:

**Comment by fcnjd** [comment]
I came here since I'm having the same issue bot with Chrome and Firefox. Maybe this info helps: When I press NVDA+space in this invisible dialog to turn off browse mode, following error is logged Can this issue please be re-opened? As @dreinn already pointed out very well, it only happens with NVDA.
```python
ERROR - scriptHandler.executeScript (12:44:17.244) - MainThread (15748):
error executing script: <bound method GlobalCommands.script_toggleVirtualBufferPassThrough of <globalCommands.GlobalCommands object at 0x06C3D470>> with gesture 'NVDA+Leerzeichen'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 300, in executeScript
  File "globalCommands.pyc", line 2314, in script_toggleVirtualBufferPassThrough
  File "treeInterceptorHandler.pyc", line 161, in _set_passThrough
  File "review.pyc", line 138, in setCurrentMode
  File "review.pyc", line 32, in getObjectPosition
AttributeError: 'NoneType' object has no attribute 'makeTextInfo'
```
